openmeter/customer/customer.go

```diff
@@ -23,6 +24,66 @@ type Customer struct {
 	External          *CustomerExternalMapping  `json:"external"`
 }
 
+// AsAPICustomer converts a Customer to an API Customer
+func (c Customer) AsAPICustomer() (api.Customer, error) {
+	customer := api.Customer{
+		Id:               &c.ManagedResource.ID,
+		Name:             c.Name,
+		UsageAttribution: api.CustomerUsageAttribution{SubjectKeys: c.UsageAttribution.SubjectKeys},
+		PrimaryEmail:     c.PrimaryEmail,
+	}
+
+	if c.BillingAddress != nil {
+		address := api.Address{
+			City:        c.BillingAddress.City,
+			State:       c.BillingAddress.State,
+			PostalCode:  c.BillingAddress.PostalCode,
+			Line1:       c.BillingAddress.Line1,
+			Line2:       c.BillingAddress.Line2,
+			PhoneNumber: c.BillingAddress.PhoneNumber,
+		}
+
+		if c.BillingAddress.Country != nil {
+			country := string(*c.BillingAddress.Country)
+			address.Country = &country
+		}
+
+		customer.BillingAddress = &address
+	}
+
+	if c.External != nil {
+		external := api.CustomerExternalMapping{}
+
+		if c.External.StripeCustomerID != nil {
+			external.StripeCustomerId = c.External.StripeCustomerID
+		}
+
+		customer.External = &external
```

Comment:
```suggestion
		customer.External = &api.CustomerExternalMapping{
		  StripeCustomerId: c.External.StripeCustomerID
		 }
```

Reply:
API response would be different. null vs empty object